packages/web-app/src/modules/auth-views/components/FramePage.tsx

```diff
@@ -101,13 +102,17 @@ export const FramePage = withStyles(styles)(
     }
 
     render() {
-      const { onCloseRequested, frameSandbox, frameTitle, frameUrl, classes } = this.props
+      const { onCloseRequested, frameSandbox, frameTitle, frameUrl, showWelcomeText, classes } = this.props
       const { frameLoaded } = this.state
       return (
         <Portal>
           <Overlay onCloseRequested={onCloseRequested} />
           <div className={classNames(classes.absoluteCenter, classes.container)}>
-            <NotMobile>{this.state.frameLoaded && <h1 className={classes.title}>Welcome to Salad!</h1>}</NotMobile>
+            {this.state.frameLoaded && showWelcomeText && (
```

Comment:
nit (consistency with the usage below):

```suggestion
            {frameLoaded && showWelcomeText && (
```